### DISCUSSION TITLE
Video Output Duration?

### DISCUSSION THREAD

**Discussion by smilinggoat** [noisedeck/noisedeck-docs#34] [Questions and Issues] [answered]
Hi there, I've been playing around with the free noisedeck and it's awesome! I'm interested in trying out the Pro version with audio input. Is the video output duration able to be configured to be longer than 3-seconds? Thanks!

**Comment by jesshewitt** [comment, marked as answer]
Hello, glad you're enjoying the free version! Yes, the video output can be set to longer than 3 seconds in the Pro version.
